app/models/vacancy.rb

```diff
@@ -46,7 +46,7 @@ class Vacancy < ApplicationRecord
   array_enum phases: { nursery: 0, primary: 1, middle: 2, secondary: 3, sixth_form_or_college: 4, through: 5 }
   array_enum job_roles: JOB_ROLES
 
-  enum :contract_type, { permanent: 0, fixed_term: 1, parental_leave_cover: 2, casual: 3 }
+  enum :contract_type, { permanent: 0, fixed_term: 1, casual: 3 }
```

Comment:
did we want to add a comment for this non-obvious change? Good to record that there are no instances of it on production?